### ISSUE TITLE
Pipe compiles to wrong semantics

### ISSUE THREAD

**Issue by cie** [DanielXMoore/Civet#933]
Hello! Awesome project!
It seems to me that this
```rescript
init()
readFile()
|> .content
|> .value
```
incorrectly compiles to
```js
init()(readFile().content).value;
```
while I would expect
```js
init();
readFile().content.value;
```

However it works correctly for these:
```rescript
init()
readFile()
|> .content
|> value

init()
readFile()
|> content
|> .value
```
```js
init();
value(readFile().content);

init();
content(readFile()).value;
```

**Comment by edemaine** [comment]
This is an ASI bug. Thanks for reporting!

**Comment by edemaine** [comment]
Should be fixed in 0.6.66. Thanks again for the report!
